annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -337,14 +385,16 @@ private IsoformOverride getIsoformOverride(String source, String transcriptId)
 
     private List<Hotspot> getHotspotAnnotation(TranscriptConsequence transcript)
     {
-        //String transcriptId = transcript.getTranscriptId();
-        //Hotspot hotspot = hotspotRepository.findOne(transcriptId);
 
+        // String transcriptId = transcript.getTranscriptId();
+        // Hotspot hotspot = hotspotRepository.findOne(transcriptId);
+
+        // hotspotService.setHotspotsURL("http://cancerhotspots.org/api/hotspots/single/");
```

Comment:
Nothing was actually modified, I just uncommented and re-commented some code when I was testing it out